op/model.py

```diff
@@ -251,8 +260,8 @@ class RelationNotFound(ModelError):
 
 class ModelBackend:
     def __init__(self):
-        self.local_unit_name = os.environ['JUJU_UNIT_NAME']
-        self.local_app_name = self.local_unit_name.split('/')[0]
+        self.our_unit_name = os.environ['JUJU_UNIT_NAME']
+        self.our_app_name = self.our_unit_name.split('/')[0]
```

Comment:
These would be better off as _unit_name_ and _app_name_ I think, analogous to how the Model has simply _unit_ and _app_. The idea, as we talked today in the call, is to not expose that "our" terminology.  People can tell it's "ours" precisely by doing _(unit is model.unit)_.